content/cloud-docs/api-docs/run.mdx

```diff
@@ -109,6 +109,7 @@ Properties without a default value are required.
 
 | Key path                                           | Type                 | Default                                            | Description                                                                                                                                                                                                                                |
 | -------------------------------------------------- | -------------------- | -------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `data.attributes.allow-empty-apply`                | bool                 | false                                              | Whether this run should execute a [no operation apply](/cloud-docs/run/modes-and-options#no-operation-apply).                                                                                                                                             |
```

Comment:
I like that we're using "no-operation" to help people understand what this means, but the actual type is called "empty apply" so I don't want to use "no-operation" as a link. What do you think of this?

```suggestion
| `data.attributes.allow-empty-apply`                | bool                 | false                                              | Whether this run should execute a no-operation apply from a plan with no changes. Refer to [empty apply](/cloud-docs/run/modes-and-options#no-operation-apply) for details.                                                                                                                                            |
```